code/def_files/data/effects/deferred-f.sdr

```diff
@@ -98,7 +98,7 @@ void GetLightInfo(vec3 position, out vec3 lightDir, out float attenuation)
 				}
 			}
 		}
-
+		attenuation*=attenuation;
```

Comment:
Just a style thing.  It's much more readable if it's `attenuation *= attenuation`. It might be helpful to think of how white space, parentheses and such make the code clearer.  You want your code to be readable by any intrepid new coder who comes down the block.

Reply:
For the record you don't have to explain the value of style fixes in general to me, I get it, I just suck at paying attention to and spotting it.